src/components/KeysHeader/styles.js

```diff
@@ -1,13 +1,37 @@
 import styled from 'styled-components'
 import { StyledButton } from '@components/Button/styles'
+import { StyledInput } from '@components/Input/styles'
 
 export const Wrapper = styled.div`
   display: grid;
-  grid-template-columns: auto 200px;
+  grid-template-columns: calc(130px + ${({ theme }) => theme.sizes.regular}) 130px auto;
   align-items: center;
   margin-bottom: ${({ theme }) => theme.sizes.regular};
 
   ${StyledButton} {
     justify-self: end;
   }
+
+  ${StyledInput} {
+    padding: 0.5em 0.7em;
+    height: auto;
+    min-width: 160px;
+
+    &::-webkit-input-placeholder {
+      font-style: italic;
+      color: ${({ theme }) => theme.colors.border};
+    }
```

Comment:
hello
welcome
hi
yes
thank you

`&::-webkit-input-placeholder, &:-moz-placeholder`

Reply:
no?